web/src/features/moreCast2/components/MoreCast2DataGrid.tsx

```diff
@@ -33,47 +33,60 @@ const MoreCast2DataGrid = (props: MoreCast2DataGridProps) => {
   const { rows } = props
   const loading = useSelector(selectMorecast2TableLoading)
 
-  const gridColumnDefGenerator = (field: string, headerName: string, precision: number) => {
-    return {
-      field: field,
-      disableColumnMenu: true,
-      disableReorder: true,
-      editable: true,
-      headerName: headerName,
-      sortable: false,
-      type: 'number',
-      width: 120,
-      valueFormatter: (params: GridValueFormatterParams) => predictionItemValueFormatter(params, precision),
-      valueGetter: (params: GridValueGetterParams) => predictionItemValueGetter(params),
-      valueSetter: (params: GridValueSetterParams) => predictionItemValueSetter(params, field)
+  const numFormatter = (value: number, precision: number) => {
+    if (isUndefined(value) || isNaN(value)) {
+      return NaN
     }
+
+    const multiplier = Math.pow(10, precision || 0)
+    return Math.round(value * multiplier) / multiplier
```

Comment:
I think this is just end-of-day brain, but what's going on here?

Reply:
The function rounds a number to the number of decimal places specified by `precision`. 

For example,
`numFormatter(3.123456789, 1)` => 3.1

So instead of displaying 3.123456789 when starting to edit the cell, we will display 3.1.